src/main/java/cz/muni/fi/dao/TeamDaoImpl.java

```diff
@@ -0,0 +1,19 @@
+package cz.muni.fi.dao;
+
+import cz.muni.fi.entities.Car;
+
+/**
+ * 
+ * @author Lucie Kureckova, 445264
+ */
+public class TeamDaoImpl implements TeamDao {
+
+    public void changeFirstCar(Car car) {
```

Comment:
na prvy milestone bude zrejme potrebne naimplementovat

Reply:
pls refactor according to updated uml